apps/server/src/modules/deletion/deletion-api.module.ts

```diff
@@ -33,6 +34,7 @@ import { DeletionRequestUc } from './uc';
 		AuthenticationModule,
 		RocketChatUserModule,
 		RegistrationPinModule,
+		FilesStorageClientModule,
```

Comment:
You don't need it here, because you import DeleteModule

Reply:
removed